Client/Frontend/Home/HistoryPanel.swift

```diff
@@ -430,3 +493,125 @@ class HistoryPanel: SiteTableViewController, HomePanel {
         return [delete]
     }
 }
+
+class SyncedTabsTableViewCell: UIView {
+    var navigationController: UINavigationController? = nil
+    weak var homePanelDelegate: HomePanelDelegate? = nil
+    var profile: Profile!
+    weak var historyPanelController: HistoryPanelViewController!
+
+    private let ImageMargin: CGFloat = 12
+
+    lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = HistoryPanelUX.SyncedTabsCellTextColor
+        label.text = Strings.SyncedTabsTableViewCellTitle
+        label.font = DynamicFontHelper.defaultHelper.DeviceFontHistoryPanel
+        return label
+    }()
+
+    lazy var descriptionLabel: UILabel? = {
+        return nil
+    }()
+
+    lazy var imageView: UIImageView = {
+        let image = UIImage(named: "panelIconSyncedTabs")!
+        return UIImageView(image: image)
+    }()
+
+    lazy var chevron: ChevronView = {
+        let chevron = ChevronView(direction: .Right)
+        chevron.tintColor = HistoryPanelUX.SyncedTabsCellTextColor
+        chevron.lineWidth = 3.0
+        return chevron
+    }()
+
+    lazy var topBorder: UIView = {
+        let view = UIView()
```

Comment:
Nit: This lazy block and the one below are the exact same code. Why not bundle it up into a small function?

```swift
lazy var topBorder = self.createBorderView()
lazy var bottomBorder = self.createBorderView()

private createBorderView() -> UIView {
    let view = UIView()
    view.backgroundColor = SiteTableViewControllerUX.HeaderBorderColor
    return view
}